### ISSUE TITLE
AdColony Adapter: UI API called on a background thread

### ISSUE THREAD

**Issue by stsandro** [googleads/googleads-mobile-ios-mediation#363]
There is a problem with the AdColony adapter at runtime. The UI is called from a background thread.

```
AdColonyAdapter/Bidding/GADMAdapterAdColonyRTBBannerRenderer.m: runtime: UI API called from background thread: -[UIViewController presentedViewController] must be used from main thread only

#1	0x0000000102b091f7 in GADViewControllerToPresentFromForViewController ()
#2	0x0000000102adea43 in ___lldb_unnamed_symbol3393$$AdsTest ()
#3	0x00000001028bcf33 in __46-[GADMAdapterAdColonyRTBBannerRenderer loadAd]_block_invoke at /Volumes/google/src/cloud/landacastro/AdColonyAdapter-ios-release-364/google3/third_party/objective_c/gma_sdk_mediation/adapters/AdColonyAdapter/AdColonyAdapter/Bidding/GADMAdapterAdColonyRTBBannerRenderer.m:75
#4	0x00000001028ba287 in __85-[GADMAdapterAdColonyInitializer initializeAdColonyWithAppId:zones:options:callback:]_block_invoke at /Volumes/google/src/cloud/landacastro/AdColonyAdapter-ios-release-364/google3/third_party/objective_c/gma_sdk_mediation/adapters/AdColonyAdapter/AdColonyAdapter/GADMAdapterAdColonyInitializer.m:68
#5	0x000000010920033b in _dispatch_call_block_and_release ()
#6	0x0000000109201b93 in _dispatch_client_callout ()
#7	0x0000000109208cc3 in _dispatch_lane_serial_drain ()
#8	0x0000000109209977 in _dispatch_lane_invoke ()
#9	0x00000001092159ff in _dispatch_workloop_worker_thread ()
#10	0x00000001cc0adb3f in _pthread_wqthread ()
```

**Comment by chlandacastro** [comment]
HI @stsandro and @Amj-94 !  This issue was fixed in will be available in the next release.
regards, 
Christian.

**Comment by stsandro** [comment]
Hi @chlandacastro 👋,
I've can verify that is fixed with the adapter version **4.9.0.1**. Many thanks 🙏
